style.css

```diff
@@ -0,0 +1,387 @@
+body{
+    margin: 0rem;
+    padding: 0rem;
+    font-family: sans-serif;
+}
+:root{
+    font-size: 10px;
+}
+.navbar{
+    height: 8.05rem;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 0.12rem solid #eee;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.logo{
+    height: 8.05rem;
+    border-right: 0.12rem solid #eeee;
+}
+.blink{
+    margin-left: 3.5rem;
+    color: #f8cb46;
+    font-size: 4.5rem;
+    font-family: inherit;
+    letter-spacing: -0.3rem;
+    font-weight: bold;
```

Comment:
you can move ` font-size: 4.5rem;
    font-family: inherit;
    letter-spacing: -0.3rem;
    font-weight: bold;` to `logo` class